.github/workflows/image_base.yml

```diff
@@ -57,9 +33,9 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           context: .
-          file: ./Dockerfile.base
+          file: ./build/Dockerfile.builder
```

Comment:
@rootfs I had added Dockerfiles to t he root of repo. If we want to keep Dockerfiles under `build` folder, pls move the `Dockerfile.builder` to `build` folder.
https://github.com/sustainable-computing-io/kepler/pull/1118/files#diff-e7cd9380abeae311f2b7f9ed2e780a7ea795597555411cc965191ca1e6cedf16

Reply:
The file is copied there, I'll remove it

Reply:
can you point which GH workflow step copies the `Dockerfile.builder` ?